services/ec2/drivers/Ec2EbsSnapshot.py

```diff
@@ -0,0 +1,29 @@
+import boto3
+import botocore
+import datetime
+
+from utils.Config import Config
+from services.Service import Service
+
+from services.Evaluator import Evaluator
+
+class Ec2EbsSnapshot(Evaluator):
+    def __init__(self, ec2Client):
+        super().__init__()
+        self.ec2Client = ec2Client
+        self.init()
+        
+    def _checkSnapshotPublic(self):
+        snapshotData = self.ec2Client.describe_snapshots(
+            OwnerIds = ["self"],
+            RestorableByUserIds = ["all"]
+        )
+        
+        publicSnapshotList = []
+        for snapshot in snapshotData['Snapshots']:
+            publicSnapshotList.append(snapshot.get('SnapshotId'))
+        
+        if len(publicSnapshotList) > 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)